backend/controllers/cellMove/selectCell.js

```diff
@@ -28,9 +28,12 @@ module.exports = ({ elite2Api, whereaboutsApi, logError }) => async (req, res) =
 
     if (req.xhr) {
       return res.render('cellMove/partials/subLocationsSelect.njk', {
-        subLocations: locationsData
-          .find(loc => loc.key.toLowerCase() === locationId.toLowerCase())
-          .children.map(loc => ({ text: loc.name, value: loc.key })),
+        subLocations:
+          locationId === 'ALL'
+            ? []
```

Comment:
Do you need the 'No location available' dropdown element here? I seem to remember that being a requirement